component/sections/pricing/variant_d.js

```diff
@@ -151,20 +177,20 @@ function VariantD({
                         By signing up, you agree to our{" "}
                         <a
                           className="text-webriq-darkblue font-bold hover:text-webriq-darkblue"
-                          href="#"
+                          href="/terms-data-policy"
                         >
                           Terms,
                         </a>
                         <a
                           className="text-webriq-darkblue font-bold hover:text-webriq-darkblue"
-                          href="#"
+                          href="/terms-data-policy"
                         >
                           Data Policy
                         </a>{" "}
                         and{" "}
                         <a
                           className="text-webriq-darkblue font-bold hover:text-webriq-darkblue"
-                          href="#"
+                          href="/cookies-policy"
```

Comment:
As well as this @mamfi-0221

Reply:
Fixed with these pull requests: [update form static sign in link](https://github.com/webriq-pagebuilder/site-template-default/pull/31)  and [add fields for the Sign In link](https://github.com/webriq-pagebuilder/sanity-packages/pull/18)